### DISCUSSION TITLE
just a few v-card api probs

### DISCUSSION THREAD

**Discussion by luggie** [vuetifyjs/vuetify#12349] [Q&A]
(very new to vue, js and the like, question might be rookie)
Trying to implement a plotly graph into vue, which works:
```
<template>
    <div>
        <v-card height=1000>
            <plotlygraph :propData="plotInfo" divId="plot" style="height: 500"></plotlygraph>
        </v-card>
  </div>
</template>
```
however when trying any other probs then height or width for v-card, they simpley won't work.
(vuetify 2.3.12)

**Comment by Renato66** [comment]
@luggie can you host that code on a [codepen](https://codepen.io/pen/?&editable=true&editors=101=https%3A%2F%2Fvuetifyjs.com%2Fen%2Fcomponents%2Fcards%2F) example? that will be more easy to make it work
